src/CommunityToolkit.Maui.MediaElement/Views/MediaManager.android.cs

```diff
@@ -747,43 +748,28 @@ public void OnShuffleModeEnabledChanged(bool shuffleModeEnabled) { }
 	public void OnSkipSilenceEnabledChanged(bool skipSilenceEnabled) { }
 	public void OnSurfaceSizeChanged(int width, int height) { }
 	public void OnTimelineChanged(Timeline? timeline, int reason) { }
+	public void OnTrackSelectionParametersChanged(TrackSelectionParameters? parameters) { }
 	public void OnTracksChanged(Tracks? tracks) { }
-	public void OnTrackSelectionParametersChanged(TrackSelectionParameters? trackSelectionParameters) { }
 
 	#endregion
+}
 
-	/// <summary>
-	/// A <see cref="BroadcastReceiver"/> that listens for updates from the <see cref="MediaControlsService"/>.
-	/// </summary>
-	sealed class UIUpdateReceiver(IExoPlayer player) : BroadcastReceiver
+[BroadcastReceiver(Enabled = true, Exported = false)]
+sealed class UIUpdateReceiver : BroadcastReceiver
+{
+	readonly NotificationService? notificationService;
+	public UIUpdateReceiver()
+	{
+		notificationService = Microsoft.Maui.Controls.Application.Current?.MainPage?.Handler?.MauiContext?.Services?.GetService<INotificationService>() as NotificationService;
```

Comment:
Please, review how you get the Service here

Reply:
Using IPlatformApplication.Current.GetServices<T>()`. This appears to do what the helper method does.